tests/unit/test_image.py

```diff
@@ -107,3 +113,54 @@ def test_load_image_with_exif_info(self):
                 image_module.IMAGE_BACKEND.set(load_backend)
                 img = image_module.load_image(image_path)
                 assert img.shape == (10, 15, 3)
+
+
+class ImageDecodeTest:
+    def generate_test_img(self, channels) -> np.ndarray:
+        return np.random.randint(low=0, high=256, size=(5, 4, channels), dtype=np.uint8)
```

Comment:
Could you refactor 2 other encoding-decoding tests to use the same common parts?